MediaRecorder.html

```diff
@@ -565,10 +619,11 @@
     <h2>Blob Event</h2>
     <div>
       <pre class="idl">
-      [ Constructor (DOMString type, BlobEventInit eventInitDict)]
-interface BlobEvent : Event {
-    readonly        attribute Blob data;
-};</pre>
+        [ Constructor (DOMString type, BlobEventInit eventInitDict)]
```

Comment:
If you're changing this anyway, I'd remove the whitespace after `[` and all the extra whitespace before `attribute`.